src/components/20-molecules/datepicker/index.js

```diff
@@ -444,14 +478,13 @@ class AXADatepicker extends NoShadowDOM {
       }
     }
 
-    if (!this.isControlled) {
-      this.initDate(
-        null,
-        year || startDate.getFullYear(),
-        month || startDate.getMonth(),
-        day || startDate.getDate()
-      );
+    if (this.isControlled) {
```

Comment:
Easier readable would imho be: 

```
if (!this.isControlled) {
  // Rest of the method...
}
```

Reply:
This is what I had before, as the diff shows. The disadvantage is more nesting/indentation, which is usually harder to read. As you say, it's a preference, but those were my reasons.